### ISSUE TITLE
Radio Control spacing inconsistency

### ISSUE THREAD

**Issue by rodrigodagostino** [WordPress/gutenberg#17929]
The following line of code in `style.css` is causing a spacing inconsistency in the Radio Control:
```
.components-radio-control {
    display: flex;
    flex-direction: column;
}
```
The only available example can be found in the Latest Posts Block -> Post Content Settings.

This is what it looks like at the moment (I've colored the bottom margin of the surrounding element with a darker orange):
![image](https://user-images.githubusercontent.com/8670793/66732264-990bb400-ee31-11e9-9922-f303fd32f1c5.png)

The expected behavior would be having those two margins overlap each other, but `display: flex` is impeding that (I couldn't figure out what that `flex: display` is doing there in the first place).

Removing those lines of code seems to solve the problem without modifying anything else:
![image](https://user-images.githubusercontent.com/8670793/66732460-69a97700-ee32-11e9-96d7-43a45f3e7662.png)

 - OS: Windows 10 64-bit
 - Browser: Firefox [DE], Opera, Chrome
 - Version: 70.0b14, 63.0.3368.94, 77.0.3865.120

- Gutenberg 6.6.0
- WordPress 5.2.3

**Comment by brezocordero** [comment]
Reproduced with Gutenberg 6.8.0
Chrome Version 77.0.3865.120 (Official Build) (64-bit)

<img width="307" alt="FlexOverlap" src="https://user-images.githubusercontent.com/8002881/68091987-17191480-fe4c-11e9-9425-b6025d93f1d5.png">

Removing the CSS rule fixes it.

Without

```
.components-radio-control {
    display: flex;
    flex-direction: column;
}
```

<img width="308" alt="WithoutFlex" src="https://user-images.githubusercontent.com/8002881/68092074-061cd300-fe4d-11e9-95f8-57aaed1e23e9.png">

**Comment by richtabor** [comment]
I'm seeing that there's a `8px` margin-bottom applied to the `.components-base-control__field`, which seems to be leading to the extra space below the control.

<img width="401" alt="Screen Shot 2019-11-11 at 4 28 19 PM" src="https://user-images.githubusercontent.com/1813435/68622558-67bdec80-04a0-11ea-875f-827841550402.png">

**Comment by rodrigodagostino** [comment]
Thanks for your comment, but that's not the problem, @richtabor . I explained it correctly in my first message. The normal behavior is having the `components-base-control`'s 24px margin and the `components-base-control__field`'s 8px margin overlap each other. If this does not happen, then the margin below goes from the correct 24px to 32px (24px + 8px).

Those lines of CSS code that quoted are the ones responsible of breaking the normal behavior. The `display: flex` does not allow for margin overlapping.

Here's more info about the subject:
[https://www.w3.org/TR/CSS2/box.html#collapsing-margins](https://www.w3.org/TR/CSS2/box.html#collapsing-margins)

**Comment by mapk** [comment]
I've got a PR for this fix, #19340.
